src/sentry/static/sentry/app/components/forms/inputField.jsx

```diff
@@ -7,9 +7,11 @@ export default class InputField extends FormField {
   constructor(props) {
     super(props);
 
-    this.state.value = (
-      props.value !== '' ? props.value : (props.defaultValue || '')
-    );
+    this.state.value = this.valueFromProps(props);
+  }
+
+  valueFromProps(props) {
+    return props.value !== '' ? props.value : (props.defaultValue || '');
```

Comment:
Is this intentionally not `props.value || props.defaultValue || ''`? If yes, maybe add a comment, this looks confusing.